webapp/src/main/java/org/apache/atlas/web/rest/DiscoveryREST.java

```diff
@@ -433,7 +434,7 @@ public AtlasSearchResult indexSearch(@Context HttpServletRequest servletRequest,
             }
             throw abe;
         } finally {
-            if(CollectionUtils.isNotEmpty(parameters.getUtmTags())) {
+            if(CollectionUtils.isNotEmpty(parameters.getUtmTags()) && parameters.getUtmTags().contains(UTM_TAG_FROM_PRODUCT)) {
```

Comment:
Wouldn't this change the metric we're currently sending? Idea is to send another metric for identifying product calls instead of changing the behavior of current metric or panel. We will add a new panel in observability dashboard for the new metric.

Reply:
Makes sense